forms-flow-theme/scss/_button.scss

```diff
@@ -129,9 +129,10 @@ text-decoration: underline;
 .btn-light:focus,
 .btn-outline-primary:focus-visible,
 .btn-outline-primary:focus,
-.btn-secondary.form-visible,
-.btn-secondary:focus {
-  outline: 3px solid $gray-100 !important;
+.btn-secondary.focus-visible,
+.btn-secondary:focus,
+.nav-link:focus-visible {
+  outline: 3px solid var(--ff-primary-200) !important;
```

Comment:
can we use directly the variable name instead of --ff-primary-200 ?

Reply:
No, $primary-200 is a custom variable that is available only as a root variable. 
at the same time, $gray-200 is available as it is a bootstrap variable.